package.json

```diff
@@ -41,8 +41,9 @@
     "webpack": "^4.0.0 || ^5.0.0"
   },
   "dependencies": {
-    "schema-utils": "^3.0.0",
-    "loader-utils": "^2.0.0"
+    "loader-utils": "^2.0.0",
+    "lodash.camelcase": "^4.3.0",
```

Comment:
Please avoid using this module, it is deprecated, we don't need extra module for simple logic

Reply:
@evilebottnawi Sorry for my delay.

I've tried to use simpler solutions like https://stackoverflow.com/a/2970667, but they can fail with [`RangeError`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RangeError).

So since [lodash.camelcase](https://www.npmjs.com/package/lodash.camelcase) is deprecated I have remplace it with [camelcase](https://www.npmjs.com/package/camelcase).

camelcase:
- It has recent activity, latest release was published 1 month ago.
- It doesn't depend on more dependencies.
- It's small https://bundlephobia.com/result?p=camelcase@6.2.0.

If still doesn't work, I can try inlining this code https://github.com/sindresorhus/camelcase/blob/master/index.js here.